schemas/test/en/test_progress_value_source_section_enabled_hub_complex.json

```diff
@@ -0,0 +1,244 @@
+                        },
+                        {
+                            "type": "Question",
+                            "id": "s1-b2",
+                            "question": {
+                                "id": "s1-b2-q1",
+                                "title": "Section 1 Question 2",
+                                "type": "General",
+                                "description": ["Shows if the answer to the previous question is not 0"],
+                                "answers": [
+                                    {
+                                        "id": "s1-b2-q1-a1",
+                                        "mandatory": true,
+                                        "label": "Enter any number",
+                                        "type": "Number"
+                                    }
+                                ]
+                            },
+                            "skip_conditions": {
+                                "when": {
+                                    "!=": [{ "source": "answers", "identifier": "s1-b1-q1-a1" }, 0]
+                                }
+                            }
+                        }
+                    ],
+                    "id": "group-1"
+                }
+            ]
+        },
+        {
+            "id": "section-2",
+            "title": "Section 2",
+            "enabled": {
+                "when": {
+                    "==": [{ "source": "progress", "selector": "section", "identifier": "section-1" }, "COMPLETED"]
+                }
+            },
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "s2-b1",
+                            "question": {
+                                "id": "s2-b1-q1",
+                                "title": "Section 2 Question 1",
+                                "description": ["This question always shows"],
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "id": "s2-b1-q1-a1",
+                                        "mandatory": true,
+                                        "label": "Enter any number",
+                                        "type": "Number"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "s2-b2",
+                            "question": {
+                                "id": "s2-b2-q1",
+                                "title": "Section 2 Question 2",
+                                "type": "General",
+                                "description": ["Shows if the answer to the previous question is not 0"],
```

Comment:
The skip condition and description of skip condition for this question is the opposite of what it actually does. It currently shows when the prev answer is 0, not not-0. 
I think a hint of the intended behaviour could also be added to question 1.

Reply:
The actual progress source testing for this test is working fine though :)

Reply:
I'm not sure what the initial intention was for these specific blocks, maybe just testing a mixture of rules, but as you pointed out it was not working. Updated the schema so that routing rules are used, so should be have as was intended now 👍

Reply:
re-opened as the description is valid for Q3 but the second question is showing regardless of answer to first question, think it just needs a description update